### DISCUSSION TITLE
Onboarding a team planning to author a new Azure Action

### DISCUSSION THREAD

**Discussion by kaverma** [Azure/actions#57] [Ideas]
Hi @azooinmyluggage, @N-Usha  can we onboard a particular team who is planning to author a new Action for Azure to use the guidelines and building blocks we are building? This will help bake the process and platform.
cc - @kmkumaran

**Comment by kamasawaddee** [comment]
> Hi @azooinmyluggage, @N-Usha  can we onboard a particular team who is planning to author a new Action for Azure to use the guidelines and building blocks we are building? This will help bake the process and platform.
> cc - @kmkumaran 

Push
